src/views/test/operations/filter.ts

```diff
@@ -0,0 +1,175 @@
+import InputFilterInteger from '../components/InputFilterInteger'
+
+import { getNumberFilterOperator, getStringFilterOperator, getDateFilterOperator, getBooleanFilterOperator } from './filterQueries'
+import { GridColTypeDef, getGridNumericColumnOperators, getGridStringOperators, GridFilterOperator } from "@material-ui/data-grid";
+
+let filterData;
```

Comment:
I need **filterData** global variable in filter.ts. Because without it I had a lot of parameters in the functions. 
Probably exist better solution pass **filterData** from react component to helper function like in my soultion like bottom
```
let filterData;
export const getFilterData = (filter) => {
    filterData = filter
}